--- cinnamon-5.0.5/cinnamon.session.in.orig	2021-08-22 21:47:17.105643324 +0200
+++ cinnamon-5.0.5/cinnamon.session.in	2021-08-22 21:47:49.202136109 +0200
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon
-RequiredComponents=cinnamon;org.cinnamon.ScreenSaver;nemo-autostart;@REQUIRED@cinnamon-killer-daemon;
+RequiredComponents=cinnamon;org.cinnamon.ScreenSaver;polkit-cinnamon-authentication-agent-1;nemo-autostart;@REQUIRED@cinnamon-killer-daemon;
 DesktopName=X-Cinnamon
 
 
--- cinnamon-5.0.5/cinnamon2d.session.in.orig	2021-07-14 16:23:43.000000000 +0200
+++ cinnamon-5.0.5/cinnamon2d.session.in	2021-08-22 21:48:01.745401490 +0200
@@ -1,6 +1,6 @@
 [Cinnamon Session]
 Name=Cinnamon (Software Rendering)
-RequiredComponents=cinnamon2d;org.cinnamon.ScreenSaver;nemo-autostart;@REQUIRED@cinnamon-killer-daemon;
+RequiredComponents=cinnamon2d;org.cinnamon.ScreenSaver;polkit-cinnamon-authentication-agent-1;nemo-autostart;@REQUIRED@cinnamon-killer-daemon;
 DesktopName=X-Cinnamon
 
 
