--- cinnamon-1.1.3/data/org.cinnamon.gschema1.xml.in	2012-01-02 12:03:30.000000000 +0000
+++ cinnamon-1.1.3/data/org.cinnamon.gschema.xml.in	2012-01-02 16:41:39.242419968 +0000
@@ -231,7 +231,7 @@
     </key>
     
     <key name="panel-launchers" type="as">
-      <default>[ 'firefox.desktop', 'gnome-terminal.desktop', 'nautilus.desktop']</default>
+      <default>[ 'mozilla-firefox.desktop', 'gnome-terminal.desktop', 'nautilus.desktop']</default>
       <_summary>Desktop files of the applications to put in the panel launchers applet</_summary>
       <_description>
         Cinnamon allows to show applications launchers on the panel.
@@ -251,7 +251,7 @@
     </key>
 
     <key name="favorite-apps" type="as">
-      <default>[ 'firefox.desktop', 'mintInstall.desktop', 'cinnamon-settings.desktop', 'xchat.desktop', 'gnome-terminal.desktop', 'nautilus.desktop' ]</default>
+      <default>[ 'cinnamon-settings.desktop', 'mozilla-firefox.desktop', 'evolution.desktop', 'empathy.desktop', 'rhythmbox.desktop', 'shotwell.desktop', 'libreoffice-writer.desktop', 'nautilus.desktop', 'gnome-documents.desktop' ]</default>
       <_summary>List of desktop file IDs for favorite applications</_summary>
       <_description>
         The applications corresponding to these identifiers
